backend/docker/prod/lambda/Dockerfile

```diff
@@ -19,22 +22,29 @@ RUN dnf -y install \
 RUN dnf install $PYTHON_VERSION
 RUN dnf -y install python3-pip python3-devel
 
-## Add source
+## Add container user group
+RUN groupadd -r ${CONTAINER_USER_GROUP} && useradd -m -r -g ${CONTAINER_USER_GROUP} ${CONTAINER_USER}
+RUN chown -R ${CONTAINER_USER}:root /usr/local
+
+## Add your source
 WORKDIR ${FUNCTION_DIR}
+RUN chown -R ${CONTAINER_USER}:root ${FUNCTION_DIR}
+
+USER ${CONTAINER_USER}
 
 # App specific requirements
-COPY backend/requirements.txt ./requirements.txt
+COPY --chown=${CONTAINER_USER}:root backend/requirements.txt ./requirements.txt
 
 # Install App requirements
 RUN /bin/bash -c "${PYTHON_VERSION} -m pip install setuptools"
```

Comment:
Why are we not adding the `--prefix /usr/local` that we add in other Dockerfiles here?

Reply:
This is fixed @dlpzx . Looks like this file is recently updated to have the setup tools line, the one i have on my local that i pulled a while ago did not seem to have it.